### PR TITLE
feat(cedarling): add an example application using python binding

### LINKED ISSUE #9434: feat(cedarling): add an example application using python binding
- [x]  Create a sample Python application (using poetry) using python binding.

### PR DESCRIPTION
**Pull request by olehbozhok**
### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  [Link](https://github.com/JanssenProject/jans/issues/9434)
  
  
closes #9434

#### Implementation Details
  Added folder `bindings\example_cedarling_python_poetry`. With example of application using `poetry`

-------------------
### Test and Document the changes
- [x] Static code analysis has been run locally and issues have been fixed
- [ ] Relevant unit and integration tests have been added/updated
- [x] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

Please check the below before submitting your PR. The PR will not be merged if there are no commits that start with `docs:` to indicate documentation changes or if the below checklist is not selected.
- [ ] **I confirm that there is no impact on the docs due to the code changes in this PR.**

### CODE REVIEW

File: jans-lock/cedarling/examples/cedarling_python_poetry/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    If you haven't already initialized your project, create a new project or initialize an existing one with Poetry:
+
+    ```bash
+    poetry init
+    ```
+
+    This command will help you set up the `pyproject.toml` file.
+
+1. Compile `whl` file for cedarling_python.
+    Watch to the `cedarling_python` folder and in [README.md](../../bindings\cedarling_python\README.md) file.
```
Comment (SafinWasi) [inline jans-lock/cedarling/examples/cedarling_python_poetry/README.md]: This path is broken on Github. Please use unix path separator `/` for paths.
image

File: jans-lock/cedarling/examples/cedarling_python_poetry/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    You need to read the `Prerequisites` and `Only Building the Library` sections. And then follow the steps.
+
+    After executing the last command `maturin build` you will see in the console something like this:
+    ```
+    ...
+    Finished `dev` profile [unoptimized + debuginfo] target(s) in 20.27s
+    📦 Built wheel for CPython 3.11 to D:\upwork\jans\jans-lock\cedarling\target\wheels\cedarling_python-0.1.0-cp311-none-win_amd64.whl
+    ```
+    Where `D:\upwork\jans\jans-lock\cedarling\target\wheels\cedarling_python-0.1.0-cp311-none-win_amd64.whl` is the path of the `whl` file.
```
Comment (SafinWasi) [inline jans-lock/cedarling/examples/cedarling_python_poetry/README.md]: Please use partial paths, absolute paths may not exist on other systems

File: jans-lock/cedarling/examples/cedarling_python_poetry/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+1. **Install Dependencies **
```
Comment (SafinWasi) [inline jans-lock/cedarling/examples/cedarling_python_poetry/README.md]: Syntax error for bold, should be `**Install Dependencies**`

File: jans-lock/cedarling/examples/cedarling_python_poetry/pythonpoetry.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    # also fields support setters and getter
+    config.policy_store = store
+
+    authz = init(config)
+
+    # Create a new Request instance
+    req = Request()
+
+    req.access_token = input("insert assess_token:")
```
Comment (SafinWasi) [inline jans-lock/cedarling/examples/cedarling_python_poetry/pythonpoetry.py]: Typo, `assess_token` should be `access_token`

File: jans-lock/cedarling/examples/cedarling_python_poetry/README.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    poetry init
+    ```
+
+    This command will help you set up the `pyproject.toml` file.
+
+1. Compile `whl` file for cedarling_python.
+    Watch to the `cedarling_python` folder and in [README.md](../../bindings\cedarling_python\README.md) file.
+    You need to read the `Prerequisites` and `Only Building the Library` sections. And then follow the steps.
+
+    After executing the last command `maturin build` you will see in the console something like this:
```
Comment (SafinWasi) [inline jans-lock/cedarling/examples/cedarling_python_poetry/README.md]: Should be `maturin build --release`

**Review by SafinWasi — CHANGES_REQUESTED** [review]
Code worked for me, but documentation needs fixing

File: jans-lock/cedarling/examples/cedarling_python_poetry/README.md (hunk 1 above)
Comment (olehbozhok) [reply jans-lock/cedarling/examples/cedarling_python_poetry/README.md]: resolved

File: jans-lock/cedarling/examples/cedarling_python_poetry/README.md (hunk 5 above)
Comment (olehbozhok) [reply jans-lock/cedarling/examples/cedarling_python_poetry/README.md]: resolved

File: jans-lock/cedarling/examples/cedarling_python_poetry/README.md (hunk 2 above)
Comment (olehbozhok) [reply jans-lock/cedarling/examples/cedarling_python_poetry/README.md]: updated with prefix
`D:\path_to_the_project`

File: jans-lock/cedarling/examples/cedarling_python_poetry/README.md (hunk 3 above)
Comment (olehbozhok) [reply jans-lock/cedarling/examples/cedarling_python_poetry/README.md]: resolved

File: jans-lock/cedarling/examples/cedarling_python_poetry/pythonpoetry.py (hunk 4 above)
Comment (olehbozhok) [reply jans-lock/cedarling/examples/cedarling_python_poetry/pythonpoetry.py]: resolved

**Review by SafinWasi — APPROVED** [review]
Looks good

**Review by moabu — APPROVED** [review]

**Pull request merged**